pkg/kubeadm/api/types.go

```diff
@@ -37,8 +37,12 @@ type KubeadmConfig struct {
 
 type InitFlags struct {
 	API struct {
-		AdvertiseAddrs  []net.IP
-		ExternalDNSName []string
+		AdvertiseAddrs        []net.IP
+		ExternalDNSName       []string
+		ExternalEtcdEndpoints []string
```

Comment:
I'd move these into another subfield. This field is meant to be only about settings that relate to "how is API server reached?".